homeassistant/components/axis/diagnostics.py

```diff
@@ -0,0 +1,45 @@
+"""Diagnostics support for Axis."""
+from __future__ import annotations
+
+from typing import Any
+
+from homeassistant.components.diagnostics import async_redact_data
+from homeassistant.config_entries import ConfigEntry
+from homeassistant.const import CONF_MAC, CONF_PASSWORD, CONF_UNIQUE_ID
+from homeassistant.core import HomeAssistant
+
+from .const import DOMAIN as AXIS_DOMAIN
+
+REDACT_CONFIG = {CONF_MAC, CONF_PASSWORD, CONF_UNIQUE_ID}
```

Comment:
There is no need to redact the entriy unique I'd, it is nothing sensitive ;)
```suggestion
REDACT_CONFIG = {CONF_MAC, CONF_PASSWORD}
```

Reply:
Mac addresses are sensitive?

Reply:
ok, that is a point, when the unique id contains the mac 👍 
Nevertheless, the username must be redacted, too.

Reply:
Done